test/git.test.js

```diff
@@ -76,10 +76,12 @@ describe('git', function () {
 
       return Git.getCommits()
       .then(function (commits) {
-        Expect(commits).to.have.length(5);
+        Expect(commits).to.have.length(6);
         Expect(commits[0]).to.have.property('type');
         Expect(commits[0]).to.have.property('category');
         Expect(commits[0]).to.have.property('subject');
+
+        Expect(commits[5].type).to.eql('breaking');
```

Comment:
As opposed to including this check on an existing test, can you add an explicit test for the breaking change logic?

Reply:
👍